src/treq/client.py

```diff
@@ -141,15 +146,18 @@ def delete(self, url, **kwargs):
         """
         return self.request('DELETE', url, **kwargs)
 
-    def request(self, method, url, **kwargs):
+    def request(self, method, url,
+                headers=None, params=None, data=None, files=list(),
+                auth=None, cookies=dict(), allow_redirects=True,
```

Comment:
Don't use a mutable default even if it happens to be not mutated presently (I think it isn't but only looked for a couple seconds).

Reply:
And this is why reviews are a great thing (and why one doesn't push in a hurry). Totally right, thank you.